File: 251118/배열의 특정 위치 찾기 3/find-specific-location-fo-array-3.cpp
#include <iostream>
using namespace std;

int main() {
    // Please write your code here.
    int arr[100];
    int point;

    for(int i = 0; i < 100; i++) {
        cin >> arr[i];
        if(arr[i] == 0) {
            point = i;
            break;
        }
    }
    cout << arr[point -3]+arr[point -2]+arr[point -1];
    return 0;
}

File: 251117/배열의 특정 위치 찾기 3/find-specific-location-fo-array-3.cs
using System;

public class Codetree
{  
    public static void Main()
    {
        // Please write your code here.
        int[] save = new int[100];
        string[] input = Console.ReadLine().Split();

        int save_size = input.Length;
        for(int i = 0; i < save_size; i++) {
            save[i] = int.Parse(input[i]);
        }

        for(int i = 0; i < save_size; i++) {
            if(save[i] == 0) {
                Console.Write(save[i-3] + save[i-2] + save[i-1]);
                break;
            }
        }

        
    }
}
